pkg/scheduler/ugm/manager.go

```diff
@@ -35,23 +35,24 @@ var m *Manager
 
 const maxresources = "maxresources"
 const maxapplications = "maxapplications"
+const wildcard = "*"
 
 // Manager implements tracker. A User Group Manager to track the usage for both user and groups.
 // Holds object of both user and group trackers
 type Manager struct {
-	userTrackers      map[string]*UserTracker
-	groupTrackers     map[string]*GroupTracker
-	userLimitsConfig  map[string]map[string]map[string]interface{} // Hold limits settings of user * queue path
-	groupLimitsConfig map[string]map[string]map[string]interface{} // Hold limits settings of group * queue path
+	userTrackers              map[string]*UserTracker
+	groupTrackers             map[string]*GroupTracker
+	userWildCardLimitsConfig  map[string]map[string]interface{} // Hold limits settings of user '*'
+	groupWildCardLimitsConfig map[string]map[string]interface{} // Hold limits settings of group '*'
```

Comment:
Simplify this: introduce a struct type that has two entries. Resource and uint64 to store the wildcard settings in. Store that struct in the map. That safes one indirection and the continuous conversion of the limit object's `map[string]string` to a resource.